src/app/components/article/Articles.js

```diff
@@ -383,7 +383,7 @@ const Articles = ({
                     ... on FactCheck {
                       id
                       title
-                      description: summary
+                      summary
```

Comment:
Looks good, I confirmed that the `<ArticleCard />` is already expecting `description` or `summary`, not only `description`.